openspec/specs/skill-lifecycle/spec.md

```
@@ -0,0 +1,129 @@
+2. **Frontmatter parse**: YAML frontmatter parsed if present (parse errors are warnings)
+3. **Name validation**: name must pass `[a-z0-9-]` rules (error on install, warning on load)
+4. **Name consistency**: directory name must match frontmatter `name` field
+5. **Trust gate**: ThirdParty skills with `allowed-tools` require `--trust` or interactive consent
+6. **Scanner check**: content scanned for injection patterns; high scores block install unless `--trust`
+7. **Lockfile write**: entry added to `skills.lock` with computed content hash
+
+## Update Flow
+
+```bash
+corvus skills update [name]
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)